x-pack/plugins/security/server/authentication/api_keys/api_keys.ts

```diff
@@ -253,10 +260,10 @@ export class APIKeys {
   }
 
   /**
-   * Tries to invalidate an API key by using the internal user.
-   * @param params The params to invalidate an API key.
+   * Tries to invalidate an API keys by using the internal user.
+   * @param params The params to invalidate an API keys.
```

Comment:
__nit:__
```suggestion
   * Tries to invalidate the API keys by using the internal user.
   * @param params The params to invalidate the API keys.
```

Reply:
__nit:__ would you mind updating all log messages in this method to mention multiple keys as well?

```ts
this.logger.debug('Trying to invalidate an API key');
``` 
----> 
```ts
this.logger.debug(`Trying to invalidate ${params.ids.length} API keys`);
```

etc.